awx/main/utils/external_logging.py

```diff
@@ -22,6 +22,21 @@ def construct_rsyslog_conf_template(settings=settings):
     spool_directory = getattr(settings, 'LOG_AGGREGATOR_MAX_DISK_USAGE_PATH', '/var/lib/awx').rstrip('/')
```

Comment:
Consider refactoring the repeated constant, `/var/lib/awx`, into a variable and using that instead of all the embedded references.